index.js

```diff
@@ -6,6 +6,7 @@ var postcss = require('postcss');
 module.exports = exports = function(options) {
   var defaults = {
     pattern: ['**/*.css', '!**/_*/*', '!**/_*'],
+    parser: undefined,
```

Comment:
If you're going to add the parser option, perhaps you should add the `syntax` and `stringifier` options as well.